lib/mayaUsd/commands/Readme.md

```diff
@@ -0,0 +1,441 @@
+| `-useAsAnimationCache` | `-uac`     | bool           | false                             | Imports geometry prims with time-sampled point data using a point-based deformer node that references the imported USD file. When this parameter is enabled, `MayaUSDImportCommand` will create a `pxrUsdStageNode` for the USD file that is being imported. Then for each geometry prim being imported that has time-sampled points, a `pxrUsdPointBasedDeformerNode` will be created that reads the points for that prim from USD and uses them to deform the imported Maya geometry. This provides better import and playback performance when importing time-sampled geometry from USD, and it should reduce the weight of the resulting Maya scene since it will bypass creating blend shape deformers with per-object, per-time sample geometry. Only point data from the geometry prim will be computed by the deformer from the referenced USD. Transform data from the geometry prim will still be imported into native Maya form on the Maya shape's transform node. **Note**: This means that a link is created between the resulting Maya scene and the USD file that was imported. With this parameter off (as is the default), the USD file that was imported can be freely changed or deleted post-import. With the parameter on, however, the Maya scene will have a dependency on that USD file, as well as other layers that it may reference. Currently, this functionality is only implemented for Mesh prims/Maya mesh nodes. |
+| `-verbose`             | `-v`       | noarg          | false                             | Make the command output more verbose. |
+| `-variant`             | `-var`     | string[2]      | none                              | Set variant key value pairs |
+
+### Return Value
+
+`MayaUSDImportCommand` will return an array containing the fullDagPath
+of the highest prim(s) imported. This is generally the fullDagPath
+that corresponds to the imported primPath but could be multiple
+paths if primPath="/".
+
+
+## `MayaUSDExportCommand`
+
+### Command Flags
+
+| Long flag                        | Short flag | Type             | Default             | Description |
```

Comment:
There are few flags missing here:
-dsp -exportDisplayColor             on
-fst -frameStride                    Float
-mfc -melPerFrameCallback           
-mpc -melPostCallback               
-msn -materialsScopeName             String

Reply:
Will look it up, I thought I had compared carefully with the help from within Maya, but I must have missed them somehow.

Reply:
I don't understand the mel callbacks: they take no argument... yet the call back seem that it should be a string?